recipes/gtsam/all/conanfile.py

```diff
@@ -67,6 +70,7 @@ class GtsamConan(ConanFile):
         "install_matlab_toolbox": False,
         "install_cython_toolbox": False,
         "install_cppunitlite": True,
+        "use_system_eigen": True,
```

Comment:
Should probably make the `self.requires("eigen/3.4.0")` conditional on this being set to `True` then as well.